cfgov/jinja2/v1/youth_employment_success/route-details.html

```diff
@@ -1,92 +1,90 @@
-{% from 'alert.html' import render as renderAlert with context %}
-<div class="content-l content-l_col-2-3 yes-route-details">
-    <p class="h4 u-mt0">
+{% from 'alert.html' import render as render_alert with context %}
+{% from 'line-item.html' import render as render_line_item with context %}
+
+<div class="content-l yes-route-details">
+  <div class="content-l_col content-l_col-1">
+    <p class="h5 u-mt0 u-mb0">
```

Comment:
Yes, I liked having the `<p>` because it is a paragraph, but I also like having fewer classes annotating the elements where possible...will think about it

Reply:
Looking at this again, I'm using a `p` element because the style is slightly different when applying the `h5` class to a `div` vs a `p`. I think the `p` is what we need to achieve consistency with the design mocks.

Reply:
👍